### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by TortleWortle**
Our units should always scale with the view width and also we should be modern at all times and use modern units.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 body {
-  margin: 0px;
+  margin: 0vw;
```
Comment (m1guelpf) [inline style.css]: Didn't we agree to remove units?

File: style.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 body {
   margin: 0px;
+  margin: 0vw;
```
Comment (m1guelpf) [inline style.css:3]: Didn't we agree to remove units?

**Review by m1guelpf — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (JipFr) [reply style.css:3]: Did we?

Comment (TortleWortle) [reply style.css:3]: I'll get right on it!

Comment (TortleWortle) [reply style.css:3]: I removed them but kept the other lines as fallbacks.

Comment (JipFr) [reply style.css:3]: Why would you need fallback?

**Comment by JipFr** [comment]
Hi, could you please remove the fallback lines?

**Comment by TortleWortle** [comment]
@CommandCoder That would break our backwards compatibility with internet explorer -200 and also cause any new registered user to instantly get flooded with emails.

**Comment by JipFr** [comment]
That may be true but do I really look like I care about Harambe's mailbox?

**Pull request merged**